compiler/rustc_trait_selection/src/traits/error_reporting/suggestions.rs

```diff
@@ -690,12 +690,14 @@ impl<'a, 'tcx> InferCtxtExt<'tcx> for InferCtxt<'a, 'tcx> {
         // Blacklist traits for which it would be nonsensical to suggest borrowing.
         // For instance, immutable references are always Copy, so suggesting to
         // borrow would always succeed, but it's probably not what the user wanted.
-        let blacklist: Vec<_> =
-            [LangItem::Copy, LangItem::Clone, LangItem::Unpin, LangItem::Sized, LangItem::Send]
+        let mut blacklist: Vec<_> =
```

Comment:
```suggestion
        let mut never_suggest_borrow: Vec<_> =
```

Reply:
I didn't notice this last time, but I'd prefer to avoid 'blacklist' terminology and to use something more precise.

Reply:
I didn't add this code. This change was added by someone else after the auto traits migration PR was merged.

I can change the wording though.

Reply:
Yeah, go ahead and change it anyway.

Reply:
Thanks!